openwisp_controller/connection/models.py

```diff
@@ -226,3 +231,13 @@ def update_config(self):
             else:
                 self.device.config.set_status_applied()
                 self.disconnect()
+
+    def save(self, *args, **kwargs):
+        super().save(*args, **kwargs)
+        if self.is_working != self._initial_is_working:
+            self.send_is_working_changed_signal()
+            self.last_attempt = timezone.now()
```

Comment:
I don't think you should add this line, because that's already performed by another method which is specialized in trying the connection to the device

Reply:
Done :+1: